libs/hbb_common/protos/message.proto

```diff
@@ -31,8 +31,7 @@ message VideoFrame {
     EncodedVideoFrames h265s = 11;
     EncodedVideoFrames vp8s = 12;
     EncodedVideoFrames av1s = 13;
-  }
-  int32 display = 14;
+  } int32 display = 14;
```

Comment:
remove code format in ` message.proto` please

Reply:
ok

Reply:
https://github.com/rustdesk/rustdesk/pull/9993/files#diff-26d052b111f4884d9e4345bcc01bab2b566825cadbeb217c4d52ac66748427cc